src/main/java/com/textmessenger/model/entity/dto/TestingWs.java

```diff
@@ -0,0 +1,13 @@
+package com.textmessenger.model.entity.dto;
+
+import lombok.Data;
+
+@Data
+public class TestingWs {
+  private String sender;
+  private String receiver;
+  private String type;
+  private MessageToFront messageToFront;
+  private DialogToFront dialogToFront;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "dialogToFront" private field. ([squid:S1068](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=175&issues=AWXEj19NlgbYnohYqVxG&open=AWXEj19NlgbYnohYqVxG)

[ISSUE_KEY]: AWXEj19NlgbYnohYqVxG